xbmc/dialogs/GUIDialogContextMenu.cpp

```diff
@@ -325,7 +330,7 @@ bool CGUIDialogContextMenu::OnContextButton(const std::string &type, const CFile
     }
     // prompt user if they want to really delete the source
     if (!CGUIDialogYesNo::ShowAndGetInput(CVariant{751}, CVariant{750}))
-      return false;
+      return false; //                     "Removing source"   "Are you sure?"
```

Comment:
I'm not a big fan of these comments. It will get forgotten if strings are changed/updated, no one will ever maintain it, and the spacing is already wierd.
I'd honestly prefer them gone, or at the very least  put them inline with what they are for, ie

```
...ShowAndGetInput(CVariant{751}, // "Removing source"
                   CVariant{750})) // "Are you sure"
```

If @garbear weighs in and says he's happy, that's fine as well

Reply:
i would prefer:
`    // prompt user if they want to really delete the source`
`    if (!CGUIDialogYesNo::ShowAndGetInput(CVariant{751}, CVariant{750}))`
as the comment already talks about the constants. but i'll give the decision to @garbear.
at that point i'm fine with anything.